product-scenarios/1-integrating-systems-that-communicate-in-heterogeneous-message-formats/01-SynapseConfigProject/01-synapseConfig/src/main/synapse-config/proxy-services/1_6_3_6_Proxy_replace_replaceBodyOfPayloadUsingSourceDefinedInlineConfReg.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<proxy name="1_6_3_6_Proxy_replace_replaceBodyOfPayloadUsingSourceDefinedInlineConfReg" startOnLoad="true" transports="http https" xmlns="http://ws.apache.org/ns/synapse">
```

Comment:
this should be renamed by removing extra "replace"

Reply:
This is done